docs/dev/table/connectors/jdbc.zh.md

```diff
@@ -31,35 +31,35 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-The JDBC connector allows for reading data from and writing data into any relational databases with a JDBC driver. This document describes how to setup the JDBC connector to run SQL queries against relational databases.
+JDBC连接器允许使用JDBC驱动向任意类型的关系型数据库读取或者写入数据。本文档描述了针对关系型数据库如何通过建立JDBC连接器来执行SQL查询。
 
-The JDBC sink operate in upsert mode for exchange UPDATE/DELETE messages with the external system if a primary key is defined on the DDL, otherwise, it operates in append mode and doesn't support to consume UPDATE/DELETE messages.
+如果一个主键定义在 DDL 中，JDBC sink 将以 upsert 模式与外部系统交换 UPDATE/DELETE 消息；否则，它将以appened 模式与外部系统交换消息且不支持消费 UPDATE/DELETE 消息。
 
-Dependencies
+
+依赖
 ------------
 
 {% assign connector = site.data.sql-connectors['jdbc'] %} 
 {% include sql-connector-download-table.zh.html 
-    connector=connector
+​    connector=connector
```

Comment:
revert the change, otherwise the doc build will fail

Reply:
Thanks! Got it!